aws_lambda_builders/workflows/nodejs_npm_esbuild/workflow.py

```diff
@@ -215,7 +216,8 @@ def get_resolvers(self):
 
     def _get_esbuild_subprocess(self, subprocess_npm, scratch_dir, osutils) -> SubprocessEsbuild:
         try:
-            npm_bin_path = subprocess_npm.run(["bin"], cwd=scratch_dir)
+            npm_bin_path_root = subprocess_npm.run(["root"], cwd=scratch_dir)
+            npm_bin_path = str(Path(npm_bin_path_root, ".bin"))
```

Comment:
how can we make sure the `npm root` command is available in all supported npm versions?

Reply:
I checked in the documentation which versions this command is supported in https://docs.npmjs.com/cli/v8/commands/npm-root. Should be all the same versions as `npm bin` + npm 9.